### PR TITLE
Fixed Output Formatting

### LINKED ISSUE #1493: Add `wemake-plain` output formatter
It should be the same as `wemake`, but without colors. Why?

When I try to save the output to file, I get this:

```
  106:29   WPS220 Found too deep nesting: 28 > 20
  [34mtry[39;49;00m:
  ^
```

And I want to have this:

```
  106:29   WPS220 Found too deep nesting: 28 > 20
  try:
  ^
```

### PR DESCRIPTION
**Pull request by 0x00zer0day**
# Fixed Output Formatting

## Checklist

- [x] I have double checked that there are no unrelated changes in this pull request (old patches, accidental config files, etc)
- [x] I have created at least one test case for the changes I have made
- [x] I have updated the documentation for the changes I have made
- [x] I have added my changes to the `CHANGELOG.md`

## Related issues
- Closes #1493

🙏 Please, if you or your company is finding wemake-python-styleguide valuable, help us sustain the project by sponsoring it transparently on https://opencollective.com/wemake-python-styleguide. As a thank you, your profile/company logo will be added to our main README which receives hundreds of unique visitors per day.

### CODE REVIEW

File: wemake_python_styleguide/formatter.py [hunk 1]
```diff
@@ -24,7 +24,7 @@
    :no-undoc-members:
 
 """
-
+from os import environ
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:27]:
```suggestion

from os import environ
```

File: wemake_python_styleguide/formatter.py [hunk 2]
```diff
@@ -24,7 +24,7 @@
...
 from collections import defaultdict
 from typing import ClassVar, DefaultDict, List
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:29]:
```suggestion
from typing import ClassVar, DefaultDict, Final, List
```

File: wemake_python_styleguide/formatter.py [hunk 3]
```diff
@@ -219,7 +224,10 @@ def _underline(text: str) -> str:
     '\x1b[4mHello!\x1b[0m'
 
     """
-    return '\033[4m{0}\033[0m'.format(text)
+    if _DISABLE_COLORS:
+        return '\033[4m{0}\033[0m'.format(text)
+    else:
+        return text
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:230]:
```suggestion
    if _DISABLE_COLORS:
        return text
    return '\033[4m{0}\033[0m'.format(text)
```

File: wemake_python_styleguide/formatter.py [hunk 4]
```diff
@@ -219,7 +224,10 @@ def _underline(text: str) -> str:
     '\x1b[4mHello!\x1b[0m'
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:224]: Please, also update doctests

File: wemake_python_styleguide/formatter.py [hunk 5]
```diff
@@ -208,7 +210,10 @@ def _bold(text: str) -> str:
     '\x1b[1mHello!\x1b[0m'
 
     """
-    return '\033[1m{0}\033[0m'.format(text)
+    if _DISABLE_COLORS:
+        return '\033[1m{0}\033[0m'.format(text)
+    else:
+        return text
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:216]:
```suggestion
    if _DISABLE_COLORS:
        return text
    return '\033[1m{0}\033[0m'.format(text)
```

File: wemake_python_styleguide/formatter.py [hunk 6]
```diff
@@ -230,11 +238,14 @@ def _highlight(source: str, lexer, formatter) -> str:
...
-        # or not available at all. In this case code will be just text.
+    if _DISABLE_COLORS:
+        try:
+            return highlight(source, lexer, formatter)
+        except Exception:  # pragma: no cover
+            # Might fail on some systems, when colors are set incorrectly,
+            # or not available at all. In this case code will be just text.
+            return source
+    else:
         return source
```
Comment (sobolevn) [inline wemake_python_styleguide/formatter.py:249]:
```suggestion
    if _DISABLE_COLORS:
        return source
    try:
        return highlight(source, lexer, formatter)
    except Exception:  # pragma: no cover
        # Might fail on some systems, when colors are set incorrectly,
        # or not available at all. In this case code will be just text.
        return source
```

**Comment by 0x00zer0day** [comment]
@sobolevn by doctests do you mean to update the docstrings in those 3 functions?

**Comment by sobolevn** [comment]
No, I mean doctests that are in docstrings :)
See https://docs.python.org/3/library/doctest.html

**Comment by 0x00zer0day** [comment]
Are these changes alright?

**Comment by 0x00zer0day** [comment]
@sobolevn Hey could you please help me here with the doctests that you mentioned before?

**Comment by sobolevn** [comment]
Will do today :)

**Comment by sobolevn** [comment]
Thanks a lot for your contribution!

**Pull request merged**